swift/.swiftpm/xcode/xcshareddata/xcschemes/FlatBuffers.xcscheme

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Ideally we don't store IDE data in repo, not sure how unavoidable that is in the case of Swift. Does this code build from the command-line without XCode files?

Reply:
I believe we can remove that indeed, however we will need a way to make SPM work since it would be a nested file. since it would be looking for the Package.swift, so I think we might need to make it a git submodule

Reply:
We currently don't have any submodules for this repo, so would be good to avoid starting that now :)